Lib/unittest/test/testmock/testmock.py

```diff
@@ -1769,6 +1769,35 @@ def test_attribute_deletion(self):
             self.assertRaises(AttributeError, getattr, mock, 'f')
 
 
+    def test_mock_does_not_raise_on_repeated_attribute_deletion(self):
```

Comment:
This doesn't seem like the right name, we appear to be testing that deleting an attribute of a mock *does* raise an AttributeError on the second attempt. `test_delete_of_deleted_raises_attribute_error`?

Reply:
I will eliminate the extra checks now that we have a second test for that.